contracts/.envrc-example

```diff
@@ -5,6 +5,7 @@ source_up_if_exists
 ## BeefyClient
 export RANDAO_COMMIT_DELAY=3
 export RANDAO_COMMIT_EXP=8
+export BEEFY_MINIMUM_SIGNATURE_SAMPLES=16
```

Comment:
```suggestion
export MINIMUM_REQUIRED_SIGNATURES=16
```

Reply:
Resolved in https://github.com/Snowfork/snowbridge/pull/956/commits/88f2b0749ba0bfde1a78ba2e9ff5e58e77b73207